server/database/data/connection.js

```diff
@@ -0,0 +1,27 @@
+const { Pool } = require('pg');
+require('dotenv').config()
+
```

Comment:
Destructure the process variables here and edit their names to these names
```suggestion
const {
  env: { NODE_ENV, TEST_DB_URL, DATABASE_URL, DEV_DB_URL },
} = process;
```